blockbase/inc/customizer/wp-customize-fonts.php

```diff
@@ -3,552 +3,178 @@
-		),
-		'playfair-display'  => array(
-			'fontFamily' => '"Playfair Display", serif',
-			'slug'       => 'playfair-display',
-			'name'       => 'Playfair Display',
-			'google'     => 'family=Playfair+Display:ital,wght@0,400..900;1,400..900',
-		),
-		'poppins'           => array(
-			'fontFamily' => '"Poppins", sans-serif',
-			'slug'       => 'poppins',
-			'name'       => 'Poppins',
-			'google'     => 'family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900',
-		),
-		'raleway'           => array(
-			'fontFamily' => '"Raleway", sans-serif',
-			'slug'       => 'raleway',
-			'name'       => 'Raleway',
-			'google'     => 'family=Raleway:ital,wght@0,100..900;1,100..900',
-		),
-		'red-hat-display'   => array(
-			'fontFamily' => '"Red Hat Display", sans-serif',
-			'slug'       => 'red-hat-display',
-			'name'       => 'Red Hat Display',
-			'google'     => 'family=Red+Hat+Display:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900',
-		),
-		'roboto'            => array(
-			'fontFamily' => '"Roboto", sans-serif',
-			'slug'       => 'roboto',
-			'name'       => 'Roboto',
-			'google'     => 'family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,800;1,900',
-		),
-		'roboto-slab'       => array(
-			'fontFamily' => '"Roboto Slab", sans-serif',
-			'slug'       => 'roboto-slab',
-			'name'       => 'Roboto Slab',
-			'google'     => 'family=Roboto+Slab:wght@100..900',
-		),
-		'rubik'             => array(
-			'fontFamily' => '"Rubik", sans-serif',
-			'slug'       => 'rubik',
-			'name'       => 'Rubik',
-			'google'     => 'family=Rubik:ital,wght@0,300..900;1,300..900',
-		),
-		'source-sans-pro'   => array(
-			'fontFamily' => '"Source Sans Pro", sans-serif',
-			'slug'       => 'source-sans-pro',
-			'name'       => 'Source Sans Pro',
-			'google'     => 'family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900',
-		),
-		'source-serif-pro'  => array(
-			'fontFamily' => '"Source Serif Pro", serif',
-			'slug'       => 'source-serif-pro',
-			'name'       => 'Source Serif Pro',
-			'google'     => 'family=Source+Serif+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900',
-		),
-		'space-mono'        => array(
-			'fontFamily' => '"Space Mono", sans-serif',
-			'slug'       => 'space-mono',
-			'name'       => 'Space Mono',
-			'google'     => 'family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700',
-		),
-		'work-sans'         => array(
-			'fontFamily' => '"Work Sans", sans-serif',
-			'slug'       => 'work-sans',
-			'name'       => 'Work Sans',
-			'google'     => 'family=Work+Sans:ital,wght@0,100..900;1,100..900',
-		),
-	);
+// Use an early priority to migrate legacy font settings before registering Google Fonts in `blockbase_register_google_fonts`.
+add_action( 'after_setup_theme', 'migrate_blockbase_custom_fonts', 0 );
```

Comment:
Note that I moved this to an earlier priority so it runs before `blockbase_register_google_fonts`. I was seeing one request (when the migration was run) where the fonts didn't load on the front end, and this seems to fix it.